app/webpacker/components/RegistrationsV2/Register/index.jsx

```diff
@@ -20,6 +20,7 @@ const editableRegistrationStates = ['accepted', 'pending', 'waiting_list'];
 export default function Index({
   competitionInfo,
   userInfo,
+  registrationId = null,
```

Comment:
why is there a default value?

Reply:
That's true, but it relies on JavaScript internals. I still think that explicitly marking the `null` default value makes it clear to someone reading the method signature that "competitionInfo and userInfo must be there, registrationId can be dropped if you want".

Reply:
Also, TypeScript.

Reply:
okay, but then add a comment that you explicitly mark this as optional and that it should be fixed when moving to typescript

Reply:
I have decided to remove the default value in all places that are React props, but keep it in the routing function call